apps/video/test/marionette/lib/video.js

```diff
@@ -263,6 +275,13 @@ Video.prototype = {
   },
 
   /**
+   * Click option button from player view and wait for button selection
```

Comment:
nit: does the function "wait for button selection" or does it simply click the options button?